package.json

```diff
@@ -62,7 +62,7 @@
     "test:ci": "node packages/build/bin/run-nyc npm run mocha --scripts-prepend-node-path",
     "verify:docs": "npm run build:site -- --verify",
     "build:site": "./bin/build-docs-site.sh",
-    "mocha": "node packages/build/bin/run-mocha \"labs/*/dist/__tests__/**/*.js\"",
+    "mocha": "node packages/build/bin/run-mocha \"packages/*/dist/__tests__/**/*.js\" \"examples/*/dist/__tests__/**/*.js\" \"packages/cli/test/**/*.js\" \"packages/build/test/*/*.js\"",
```

Comment:
I reverted the mocha script change in previous commits here because:

- There is no package in `/labs` yet, so running tests from that directory results in a mocha error
- It's always good to make sure production packages pass the tests before start working on the experimental feature.

The feature owner can run script to update the command instead.